### PR TITLE
New api client

### PR DESCRIPTION
**Pull request by Lrnk**
embracing change

### CODE REVIEW

File: server/jobs/page.js [hunk 1]
```diff
@@ -30,6 +34,11 @@ Page.prototype.init = function(pageId, contentArea, noBody) {
 	this.poller.start({
 		initialRequest: true
 	});
+
+	this.poller.on('error', function (err) {
+		console.log('Polling error:', err);
```
Comment (Lrnk) [inline server/jobs/page.js]: This is probably an obvious question, but should Iin fact be aiming to use next express logger in place on console.log in all cases?

Comment (matthew-andrews) [inline server/jobs/page.js]: ^^ yep :+1:  i also recommend using `name=value` type patterns to make splunk search easier, e.g.

`event=POLLING_ERROR error=${err}`

Comment (ironsidevsquincy) [inline server/jobs/page.js]: on the server, yes https://github.com/Financial-Times/next-logger

File: server/jobs/page.js [hunk 2]
```diff
@@ -2,26 +2,30 @@
...
 						return article.id;
-					}))
+					}),
+					index: 'v3_api_v2'
+				};
+
+				ft(opts)
 					.then(function (articles) {
-						console.log('fetched' + articles.length + ' articles from ' + pageId);
+						console.log('fetched ' + articles.length + ' article' + (articles.length === 1 ? '' : 's') + ' from ' + pageId);
```
Comment (matthew-andrews) [inline server/jobs/page.js]: pedant ;p

File: .gitignore [hunk 3]
```diff
@@ -10,6 +10,7 @@ npm-debug.log
 /public/*.css
 /public/*.js
 /public/*.map
+/public/*.json
```
Comment (ironsidevsquincy) [inline .gitignore]: I think it's save to just ignore everything in public

**Comment by matthew-andrews** [comment]
great to :pear: code with you, :+1:

File: server/jobs/page.js (hunk 1 above)
Comment (ironsidevsquincy) [inline server/jobs/page.js]: if the second parameter is an object, the properties will automatically turn into splunk `name-value` pairs

File: server/jobs/page.js (hunk 2 above)
Comment (Lrnk) [inline server/jobs/page.js]: :dancer:

File: server/jobs/page.js (hunk 1 above)
Comment (Lrnk) [inline server/jobs/page.js]: cool cheers guys

**Pull request merged**